pom.xml

```diff
@@ -277,7 +277,7 @@
             <dependency>
                 <groupId>org.apache.arrow</groupId>
                 <artifactId>arrow-bom</artifactId>
-                <version>15.0.2</version>
+                <version>16.0.0</version>
```

Comment:
let's wait for 16.0.1

Reply:
Why?

Reply:
0.0 is probably shit

Reply:
Doubt that, given the fact that we test arrow in BigQuery connector :) The 16.0.0 doesn't mean that they've rewrote or change the API of the java library but merely that they've added new features that were not there before